docs/forms.md

```diff
@@ -0,0 +1,145 @@
+# OneBlink SDK | Forms
+
+## Constructor
+
+| Parameter | Required | Type | Description
+|---|---|---|---|
+| `options.accessKey` | Yes | `string` | Access key provided by OneBlink. |
+| `options.secretKey` | Yes | `string` | Secret key provided by OneBlink. |
+| `options.formsRendererOrigin` | No | `string` | The Origin of the Forms Renderer. Should only be used for development and testing purposes. |
+| `options.oneBlinkAPIOrigin` | No | `string` | The Origin of the OneBlink API. Should only be used for development and testing purposes. |
+
+### Example
+
+```javascript
+const OneBlink = require('@oneblink/sdk')
+
+const options = {
+  accessKey: '123455678901ABCDEFGHIJKL',
+  secretKey: '123455678901ABCDEFGHIJKL123455678901ABCDEFGHIJKL'
+}
+const forms = new OneBlink.Forms(options)
+```
+
+## Functions
+
+### `generateFormUrl(formId, externalId)`
+
+### Parameters
+
+| Parameter | Required | Type | Description
+|---|---|---|---|
+| `formId` | Yes | `number` | The exact id of the form you wish to generate a URL for |
+| `externalId` | No | `string` | The external identifier of the form or form submission you wish to use, this will be given back to you in the query string |
```

Comment:
of the form or form submission? Should it say something more along lines of this id will be returned to you with the submissionId for later retrieving the submission data?

Reply:
👍